pkg/server/interceptors.go

```diff
@@ -43,12 +46,27 @@ func unaryInterceptor(ctx context.Context, req interface{}, info *grpc.UnaryServ
 	return resp, err
 }
 
-func combineInterceptors(a, b grpc.UnaryServerInterceptor) grpc.UnaryServerInterceptor {
-	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
-		return a(ctx, req, info, func(actx context.Context, areq interface{}) (interface{}, error) {
-			return b(actx, areq, info, func(bctx context.Context, breq interface{}) (interface{}, error) {
-				return handler(bctx, breq)
-			})
-		})
-	}
+// the always-yes logging decider function (because grpc_zap requires a decider function)
+func yesLogIt(ctx context.Context, fullMethodName string, servingObject interface{}) bool {
+	return true
+}
+
+// WithUnaryLoggingInterceptor combines interceptors for logging all GRPC unary calls with the
+// given other unary interceptors.
+func WithUnaryLoggingInterceptor(log *zap.Logger, otherInterceptors ...grpc.UnaryServerInterceptor) grpc.UnaryServerInterceptor {
+	interceptors := append([]grpc.UnaryServerInterceptor{
+		grpc_ctxtags.UnaryServerInterceptor(),
+		grpc_zap.PayloadUnaryServerInterceptor(log, yesLogIt),
```

Comment:
i feel like we might want a different zap.Logger here so that these new grpc request logs can be written somewhere else and not mixed in with normal application logs.